ilc/server/types/RegistryConfig.ts

```diff
@@ -1,5 +1,34 @@
 // TODO finish with all values
 
+export type Slot = {
+    appName: string;
+    kind: 'primary' | 'essential' | 'regular' | null;
+    props: {};
+};
+
+export type Route = {
+    slots: Record<string, Slot>;
+    meta: {};
+    routeId?: number;
+    route: string;
+    next: boolean;
+    template?: string;
+    orderPos: number;
+    versionId: string;
+    domain?: string;
+};
+
+export type SpecialRoute = {
```

Comment:
Maybe it makes sense to introduce some common parent interface for Route and SpecialRoute?